### PR TITLE
adds alert for 1.2.0 release

### PR DESCRIPTION
**Pull request by stockholmux**
Signed-off-by: Kyle Davis <kyledvs@amazon.com>

### Description
Top-of-website alert for 1.2.0

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _data/alert.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+message: "[OpenSearch 1.2.0 is live 🍾 Try the new observability interace, branding customizer, and more!](/downloads.html)"
```
Comment (elfisher) [inline _data/alert.yml:1]: interace -> interface

Otherwise looks great! Thanks!

**Review by elfisher — CHANGES_REQUESTED** [review]
one small adjustment, otherwise looks great!

File: _data/alert.yml (hunk 1 above)
Comment (stockholmux) [reply _data/alert.yml:1]: that's what I get for drafting messages in a text editor w/o spell check 🤦‍♂️

Thanks!

**Review by elfisher — APPROVED** [review]
LGTM Thanks!

**Pull request merged**